src/sentry/feedback/usecases/create_feedback.py

```diff
@@ -168,6 +168,16 @@ def create_feedback_issue(event, project_id, source: FeedbackCreationSource):
     if not project.flags.has_feedbacks:
         first_feedback_received.send_robust(project=project, sender=Project)
 
+    if (
+        source
+        in [
+            FeedbackCreationSource.NEW_FEEDBACK_ENVELOPE,
+            FeedbackCreationSource.NEW_FEEDBACK_DJANGO_ENDPOINT,
+        ]
+        and not project.flags.has_new_feedbacks
+    ):
```

Comment:
cc @JoshFerge I added `NEW_FEEDBACK_DJANGO_ENDPOINT` into here as well

Reply:
cool, sounds good. plan is to eventually deprecate that django endpoint but helpful to have here while it isn't yet.